projects/js-packages/social-logos/package.json

```
@@ -34,10 +34,14 @@
 		"react-dom": "18.3.1"
 	},
 	"devDependencies": {
+		"@types/react": "18.3.3",
+		"@types/react-dom": "18.3.0",
+		"glob": "10.4.1",
 		"svg2ttf": "^6.0.3",
 		"svgicons2svgfont": "^12.0.0",
 		"svgo": "^3.3.2",
 		"svgstore-cli": "^2.0.1",
+		"typescript": "5.4.5",
```

If we don't need anything in the newer versions, I'd suggest matching the versions already in the monorepo, to avoid adding a bunch of extra entries into the lock file.
```suggestion
		"@types/react": "18.3.1",
		"@types/react-dom": "18.3.0",
		"glob": "10.3.15",
		"svg2ttf": "^6.0.3",
		"svgicons2svgfont": "^12.0.0",
		"svgo": "^3.3.2",
		"svgstore-cli": "^2.0.1",
		"typescript": "5.0.4",
```
(After changing that, you'll want to `git checkout trunk pnpm-lock.yaml` before doing `pnpm install` to get the cleanest resulting set of updates to the lock file)

If we do need newer versions of something, let me know and we can have Renovate update it for the whole monorepo easily enough.

---

Done. Thanks